refarch-backend/src/main/resources/logback-spring.xml

```diff
@@ -8,14 +8,16 @@
     <springProperty scope="context" name="springAppName" source="spring.application.name"/>
 
 
-    <springProfile name="local,test">
+    <springProfile name="!json-logging">
 
         <!-- Log appender -->
         <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
             <encoder>
+                <!-- @formatter:off -->
```

Comment:
Which formatter gets disabled by this line? Is it the IDE internal formatting or Spotless?

Reply:
At least IDE. Not really sure if Spotless too.
Is needed as IDE is trying to wrap.

Reply:
Fine. :)